manifests/example42.pp

```diff
@@ -6,7 +6,7 @@
 class zabbix_agent::example42 {
```

Comment:
I can get rid of puppi / firewall / monitor dependencies.
Basically we can remove this whole class and where is included